stingray/gti.py

```diff
@@ -1020,6 +1079,51 @@ def time_intervals_from_gtis(gtis, chunk_length, fraction_step=1,
     return spectrum_start_times, spectrum_start_times + chunk_length
 
 
+def get_segment_bin_start(startbin, stopbin, nbin, fraction_step=1):
+    """Get start of intervals of equal length, given bin number range
```

Comment:
I don't really understand from this sentence what this function does. Maybe expand a bit?

Reply:
I think I fixed it